elements/pfe-sass/variables/_2-crayon.scss

```diff
@@ -1,11 +1,12 @@
+@import "../../../node_modules/@patternfly/patternfly/sass-utilities/colors";
```

Comment:
Needs gulp includePaths 
https://stackoverflow.com/questions/32265121/gulp-sass-work-around-for-load-path-support